Sources/SwiftCompilerPluginMessageHandling/CompilerPluginMessageHandler.swift

```diff
@@ -195,6 +224,10 @@ public class CompilerPluginMessageHandler<Provider: PluginProvider> {
   }
 }
 
+@_spi(PluginMessage)
+@available(*, deprecated, renamed: "PluginProviderMessageHandler")
+public typealias CompilerPluginMessageHandler<Provider: PluginProvider> = PluginProviderMessageHandler<Provider>
```

Comment:
Is anything in this repo or the compiler repo using this? If not, `deprecated` is not needed. Let's just remove it.

Reply:
Re-added this since it's used by `InProcPluginServer` as of https://github.com/swiftlang/swift/pull/73725. Should be a simple find-and-replace once this PR is merged.

Reply:
Ah, my bad. Thanks!